### PR TITLE
Japanese 106 keyboard map addition request

### PR DESCRIPTION
**Pull request by denkuOne**
HID map for Japanese 106 keyboard (untested)

Please add this to the keyboard list. I tried to add it myself but the app crashes and resets when I try to select the keyboard... I likely missed something in the .ino file.

### CODE REVIEW

**Comment by VoidNoi** [comment]
I'm checking it out. In the meantime please change each instance of en_JP so it specifies the corresponding [country code](https://www.loc.gov/standards/iso639-2/php/code_list.php) which I believe is "ja" so it should be: "ja_JP"

**Comment by VoidNoi** [comment]
Ok I don't think you did anything wrong. The firmware just crashes when there's more than 12 values on the kb layout menu. I'll see what I can do to fix this...

**Comment by VoidNoi** [comment]
It's working for me now, I'll merge when you finish changing the country codes :)

File: src/USBHID-Keyboard/KeyboardLayout_JP.h [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+ */
+
+#include "KeyboardLayout.h"
+
+#pragma once
+
+//Additional keys
+#define KEY_J_YEN (136+0x89)
+
+#define AMOUNT_OF_SPECIAL_CHARS 2
```
Comment (VoidNoi) [inline src/USBHID-Keyboard/KeyboardLayout_JP.h]: There is only 1 special char in the list

Comment (denkuOne) [reply src/USBHID-Keyboard/KeyboardLayout_JP.h]: I fixed the language code and special char count.

**Comment by VoidNoi** [comment]
Great! LGTM

**Pull request merged**